oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -126,6 +126,10 @@ impl Node for GrpcServerNode {
         "grpc-server"
     }
 
+    fn get_privilege(&self) -> NodePrivilege {
+        NodePrivilege::top_privilege()
```

Comment:
Could you add a comment about why this needs this privilege?

Reply:
Done.

Reply:
I would have also mentioned that this is because we don't have a more restrictive privilege that corresponds to just the user tags for now, anyways I think we have discussed it enough in issues and slack, so perhaps fine to omit.

Reply:
Added a to-do comment.